### PR TITLE
Backport trivial pr21

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/net_processing.cpp [hunk 1]
```diff
@@ -4536,7 +4536,7 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
         }
 
         //
-        // Message: getdata (non-blocks)
+        // Message: getdata (transactions)
```
Comment (UdjinM6) [inline src/net_processing.cpp:4539]: 15451: not true for Dash, should be DNM

File: test/functional/test_framework/test_framework.py [hunk 2]
```diff
@@ -103,9 +104,9 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
 
     def __init__(self):
         """Sets test framework defaults. Do not override this method. Instead, override the set_test_params() method"""
-        self.chain = 'regtest'
-        self.setup_clean_chain = False
-        self.nodes = []
+        self.chain: str = 'regtest'
```
Comment (UdjinM6) [inline test/functional/test_framework/test_framework.py:107]: 20954: requires 18210

File: src/init.cpp [hunk 3]
```diff
@@ -745,6 +745,7 @@ void SetupServerArgs()
     gArgs.AddArg("-daemon", "Run in the background as a daemon and accept commands", false, OptionsCategory::OPTIONS);
 #else
     hidden_args.emplace_back("-daemon");
+    hidden_args.emplace_back("-daemonwait");
```
Comment (UdjinM6) [inline src/init.cpp:748]: 21447: we don't have this yet

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by PastaPastaPasta** [comment]
dropped all

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**